### ISSUE TITLE
Add Post Kinds to be edited in Quick Edit

### ISSUE THREAD

**Issue by xavierroy** [dshanske/indieweb-post-kinds#114]
Allow adding or modifying post kinds from the Quick Edit option in the All Posts page.  I could push out a quick status post and later re-categorise it under a different Post Kind option.

**Comment by dshanske** [comment]
Can't promise a timeline, but this request is one I will eventually do.

**Comment by xavierroy** [comment]
I found that setting "'show_in_quick_edit' => true," in class-kind-taxonomy.php does this. :-)

**Comment by dshanske** [comment]
You are correct, however it doesn't allow for radio button selection of taxonomies, which is why it is still an issue.

**Comment by xavierroy** [comment]
Not having a radio button is ok for me as I would sometimes categorise a post with 2 or more kinds, say a check-in and an photo kind and an eat/drink for a post at a restaurant. 😀

**Comment by dshanske** [comment]
Multiple Kinds in a single post is a feature I decided not to implement.

**Comment by dshanske** [comment]
When you change the kind it now changes the mf2 data. But still no polished UI for it.

**Comment by dshanske** [comment]
Due to the fact @xavierroy is forking for this feature, I'm going to make a way to easily turn it on, even though in my opinion it isn't ready for most people.
